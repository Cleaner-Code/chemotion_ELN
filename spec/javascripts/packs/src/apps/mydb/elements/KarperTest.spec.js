import ResearchPlanFactory from 'factories/ResearchPlanFactory';
import sinon from 'sinon';
import toMatchSnapshot from 'expect-mocha-snapshot';
import expect from 'expect';
import PseudoRandomUUIDGenerator from 'factories/PseudoRandomUUIDGenerator';
import Element from '../../../../../../../app/packs/src/models/Element';

describe('KarperTest', () => {
  it('gekarpert', async function () {
    const rp1 = await ResearchPlanFactory.build('with_not_image_body_field', {}, { reset: true });

    expect.extend({ toMatchSnapshot });
    expect(rp1).toMatchSnapshot(this);
  });
});
