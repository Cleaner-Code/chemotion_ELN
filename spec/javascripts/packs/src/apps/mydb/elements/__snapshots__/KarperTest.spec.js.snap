// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KarperTest gekarpert 1`] = `
ResearchPlan {
  "_checksum": "381cecb51ca65b04b6c01c07b6e4d57c35a24ca5b23c3f8bdc93e74302970473",
  "_mode": "view",
  "_segments": Array [],
  "_wellplates": Array [],
  "attachments": Array [
    Attachment {
      "_checksum": "fabb2bc1d9ec2f5a00311079a01084d396cba323e6b548e56a92b23e602723a0",
      "id": "72d311df-7d1a-52c4-a0e9-b146640ecc2d",
      "identifier": "72d311df-7d1a-52c4-a0e9-b146640ecc2d",
      "is_new": true,
    },
  ],
  "body": Array [
    Object {
      "id": "entry-002",
      "type": "no-image",
      "value": Object {},
    },
  ],
  "can_update": true,
  "changed": false,
  "collection_id": undefined,
  "container": Container {
    "_checksum": "fabb2bc1d9ec2f5a00311079a01084d396cba323e6b548e56a92b23e602723a0",
    "attachments": Array [],
    "children": Array [
      Container {
        "_checksum": "d37525ab89c4d8c38b2d652acfa24f38b375049a501204fa716b9c9d5c086ccc",
        "attachments": Array [],
        "children": Array [],
        "container_type": "analyses",
        "description": "",
        "extended_metadata": Object {
          "report": true,
        },
        "id": "44b84426-2bc0-5462-a424-1639b7c49a79",
        "is_deleted": false,
        "is_new": true,
        "name": "new",
      },
    ],
    "container_type": "root",
    "description": "",
    "extended_metadata": Object {
      "report": true,
    },
    "id": "fc5a3a8c-f728-58ea-8d41-f9e6f1103f25",
    "is_deleted": false,
    "is_new": true,
    "name": "new",
  },
  "id": "72d311df-7d1a-52c4-a0e9-b146640ecc2d",
  "is_new": true,
  "name": "New Research Plan",
  "type": "research_plan",
}
`;
